packages/jaeger-ui/src/components/TracePage/TracePageHeader.js

```diff
@@ -170,38 +179,74 @@ export function TracePageHeaderFn(props: TracePageHeaderProps) {
     },
   ];
 
+  const embedComponent = (
```

Comment:
Seems like this can be switched to a separate component? Right now all of the JSX is being calculated and then ignored most of the time.